coreutils/Makefile

```diff
@@ -297,7 +297,10 @@ ${COREUTILS_NOLINK_DIR}:
 	echo -n "$$PROGRAM_OPTIONS" > $$OUTPUT_DIR/ProgramOptions.txt ; \
 	# Delete the sandbox
 	rm -rf sandbox sandbox.temps
-	${CURDIR}/../utils/coreutilscov.sh "$*" "$$OUTPUT_DIR" ${COREUTILS_COV_MAX_TIME}
+	# Generate line coverage information
+	if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \
+		${CURDIR}/../utils/coreutilscov.sh "$*" "$$OUTPUT_DIR" ${COREUTILS_COV_MAX_TIME}
+	fi
```

Comment:
@rasoolmaghareh The "fi" should be placed after Line 307 since the .gcda files being removed in Lines 304-307 are part of coverage computation.